third_party/spirv-tools/CMakeLists.txt

```diff
@@ -79,7 +79,7 @@ endif()
 option(SPIRV_BUILD_COMPRESSION "Build SPIR-V compressing codec" OFF)
 
 option(SPIRV_WERROR "Enable error on warning" OFF)
-if(("${CMAKE_CXX_COMPILER_ID}" MATCHES "GNU") OR ("${CMAKE_CXX_COMPILER_ID}" MATCHES "Clang"))
+if(("${CMAKE_CXX_COMPILER_ID}" MATCHES "GNU") OR (("${CMAKE_CXX_COMPILER_ID}" MATCHES "Clang") AND (NOT CMAKE_CXX_SIMULATE_ID STREQUAL "MSVC")))
```

Comment:
Not marked with `# Begin Filament changes` because this is taken from spriv upstream